### PR TITLE
Make Embed to join/leave messages

### PR DESCRIPTION
**Pull request by wicivo**
#5 

This was done using ChatGPT and I'm not sure if the code is correct, but it works.

### CODE REVIEW

**Comment by wicivo** [comment]
https://github.com/wicivo/VelocityDiscord/actions/runs/9707782745
You can download it here

File: src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java [hunk 1]
```diff
@@ -55,11 +56,10 @@ public Discord(ProxyServer server, Logger logger, Config config) {
     var messageListener = new MessageListener(server, logger, config);
 
     var builder = JDABuilder.createDefault(config.bot.DISCORD_TOKEN)
-      // this seems to download all users at bot startup and keep internal cache updated
-      // without it, sometimes mentions miss when they shouldn't
```
Comment (voruti) [inline src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java:59]: Why delete these comments?

File: src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java [hunk 2]
```diff
@@ -145,11 +145,11 @@ public void onPlayerChat(PlayerChatEvent event) {
...
       var message = new StringTemplate(config.discord.MESSAGE_FORMAT.get())
-        .add("username", username)
-        .add("server", server)
-        .add("message", content).toString();
+              .add("username", username)
+              .add("server", server)
+              .add("message", content).toString();
 
-      sendMessage(message);
+      webhookClient.sendMessage(message);
```
Comment (voruti) [inline src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java:152]: I'm unsure what this change exactly does. Can you elaborate?

**Review by voruti — CHANGES_REQUESTED** [review]
I added some comments to things I noticed in the code.

File: src/main/resources/config.toml [hunk 3]
```diff
@@ -37,7 +37,7 @@ use_webhook = false
 webhook_url = ""
 # Full URL of an avatar service to get the player's avatar from
 # Placeholders {uuid} and {username} are available
-avatar_url = "https://crafatar.com/avatars/{uuid}?overlay"
+avatar_url = "https://cravatar.eu/helmavatar/{username}"
```
[the remark this answers is not in the document]
Comment (wicivo) [reply src/main/resources/config.toml:40]: > Please explain this change. Doesn't the old url work anymore?

It didn't work for me

File: src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java (hunk 2 above)
Comment (wicivo) [reply src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java:152]: > I'm unsure what this change exactly does. Can you elaborate?

I'm not a developer and don't understand anything about code. Idea intellij showed a bug and suggested me to do this to fix it.

File: src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java [hunk 4]
```diff
@@ -213,17 +213,23 @@ public void onDisconnect(DisconnectEvent event) {
...
+      sendEmbedMessage(message.toString(), username, false);
     }
 
     updateActivityPlayerAmount();
   }
 
-  public void sendMessage(@Nonnull String message) {
-    activeChannel.sendMessage(message).queue();
+  public void sendEmbedMessage(@Nonnull String message, String username, boolean isJoin) {
+    int color = isJoin ? 0x00FF00 : 0xFF0000; // Зеленый для присоединения, Красный для выхода
```
[the remark this answers is not in the document]
Comment (wicivo) [reply src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java:226]: > Please use english in comments.

// Green to join the game, Red to leave the game

This comment was also not written by me, but by ChatGPT.

File: src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java (hunk 1 above)
Comment (wicivo) [reply src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java:59]: > Why delete these comments?

It was removed by ChatGPT

File: src/main/resources/config.toml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ceddicedced) [reply src/main/resources/config.toml:40]: The old url does indeed not work anymore.

**Pull request closed without merging**